src/views/accounts/Account/AccountList.vue

```diff
@@ -24,7 +24,12 @@ export default {
       tableConfig: {
         url: '/api/v1/accounts/accounts/',
         hasLeftActions: true,
-        hasImport: true
+        hasImport: true,
+        columnsMeta: {
+          actions: {
+            width: this.$i18n.locale === 'pt-br' ? '120px' : '100px'
+          }
+        }
       },
       treeSetting: {
         showMenu: false,
```

Comment:
There are no significant differences that need to be checked. The text appears to be formatted with appropriate indentation and spacing, without any syntax errors. No issues nor opportunities for optimization were found in the provided text snippet.